packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -232,11 +234,13 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection impleme
                 //    404: Invalid document. The document \"local/w1-...\" does not exist
                 // But this has to stay not-retriable:
                 //    406: Unsupported client protocol. This path is the only gatekeeper, have to fail!
+                //    409: Epoch Version Mismatch. Client epoch and server epoch does not match, so app needs
+                //         to be refreshed.
                 // This one is fine either way
                 //    401/403: Code will retry once with new token either way, then it becomes fatal - on this path
                 //         and on join Session path.
                 //    501: (Fluid not enabled): this is fine either way, as joinSession is gatekeeper
-                if (errorObject.code === 400 || errorObject.code === 404) {
+                if (errorObject.statusCode === 400 || errorObject.statusCode === 404) {
```

Comment:
Good catch. Maybe change it to 
> } catch (errorObject: DriverError) {
To get help from compiler?
It's a bit a lie, as it might be anything, but in cases where it's our object, it has to conform to interface.

Reply:
Actually the interface does not have statusCode property. Maybe we can work on that later i.e. adding statusCode in interface. For now this should be ok.

Reply:
Makes sense. Yes, I tried to hide it (and leave it only for telemetry) as I really do not want driver-specific error codes in host / loader layer - they need to work with errorTypes (some of them are driver-specific, but most - not).